modules/oe_multilingual_url_suffix/oe_multilingual_url_suffix.module

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Contains hooks to manipulate language links for selection page.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationUrlFallback;
+use Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix;
+
+/**
+ * Implements hook_language_types_info_alter().
+ *
+ * For more info:
+ * https://www.drupal.org/docs/8/multilingual/enable-language-negotiation.
+ *
+ * @see \language_language_types_info_alter()
+ */
+function oe_multilingual_url_suffix_language_types_info_alter(array &$language_types) {
+  $language_types[LanguageInterface::TYPE_URL]['fixed'] = [LanguageNegotiationUrlSuffix::METHOD_ID, LanguageNegotiationUrlFallback::METHOD_ID];
```

Comment:
Comment is updated

Reply:
Are we sure we need this at all? It looks like setting default state for core negotiation methods, but since we will anyway have people installing this module manually I don't see the usage for it.